advanced-operations/kubectl-plugin.md

```diff
@@ -0,0 +1,192 @@
+# Mayastor kubectl plugin
```

Comment:
Is this content different from the ones in PR https://github.com/openebs/mayastor-docs/pull/86

Reply:
The content is the same but this PR reflects this section as it has been moved under 'Advanced operations'. It was earlier under reference. The structural update (of entire TOC) can be seen in this PR and  content update in #86 .